misc/dataframeclustering/data/scripts/dataframeclustering.py

```diff
@@ -0,0 +1,70 @@
+
+import numpy as np
+import math
+
+from sklearn.preprocessing import RobustScaler
+
+def remapLables(labels):
+    """
+    Changes the label indices based on the size of each cluster, such that the 
+    cluster with most members the lowest index (zero), and the cluster with the 
+    least members gets the hightest index.
+    Negative label indices are left unchanged. (used, for example, to indicate noise in DBSCAN)
+    """
+    res = np.copy(labels)
+    label, count = np.unique(labels,return_counts=True)
+    freq = list(zip(label,count))
+    firstPostive = np.argmax(label > -1)
+    
+    for i,(l,f) in enumerate(sorted(freq[firstPostive:], key=lambda x: x[1], reverse=True)):
+        res[labels == l] = i
+    return res
+        
+
+df = np.zeros([dataframe.rows,len(activeColumnHeaders)])
```

Comment:
are the global variables `dataframe`, `activeColumnHeaders`, and `method` provided by the environment? Maybe add comment at the top